npm/qsharp/ux/circuit-vis/contextMenu.ts

```diff
@@ -311,6 +311,39 @@ const _createInputPrompt = (
   const buttonsContainer = document.createElement("div");
   buttonsContainer.classList.add("prompt-buttons");
 
+  // Create the OK button
+  const okButton = document.createElement("button");
+  okButton.classList.add("prompt-button");
+  okButton.textContent = "OK";
+
+  // Function to replace "pi" with "π" (case-insensitive)
+  const replacePiWithSymbol = (input: string) => input.replace(/pi/gi, "π");
```

Comment:
what does the `/gi` mean here? Is that regex handling for case insensitivity or is the string "gi" actually important somehow?

Reply:
The `g` means "global" which means that it will match all instances of the pattern in the string, not just the first one. The `i` does as you suggest, case insensitivity.